From e8434b9081536a50179cdc3840829f3cfac40a13 Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Wed, 19 Aug 2015 12:03:44 +0800
Subject: [PATCH] MLK-11384-1 ARM: imx: add the pcie power on contol

Regarding to the lpsr mode on imx7d arm2 board
add the pcie ep power on/off control

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index aaa9367..b8d9d00 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -709,6 +709,7 @@
 
 		pinctrl_pcie: pciegrp {
 			fsl,pins = <
+				MX7D_PAD_SAI2_TX_SYNC__GPIO6_IO19	0x2
 				MX7D_PAD_SAI2_RX_DATA__GPIO6_IO21	0x2
 			>;
 		};
@@ -931,7 +932,8 @@
 &pcie {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_pcie>;
-	reset-gpio = <&gpio6 21 0>;
+	reset-gpio = <&gpio6 21 GPIO_ACTIVE_LOW>;
+	power-on-gpio = <&gpio6 19 GPIO_ACTIVE_HIGH>;
 	status = "disabled";
 };
 
-- 
1.8.0

